x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/components/config_form.tsx

```diff
@@ -42,55 +63,184 @@ export const AgentConfigForm: React.FunctionComponent<Props> = ({
+    label: JSX.Element;
+    placeholder: string;
+  }> = useMemo(() => {
+    return [
+      {
+        name: 'name',
+        label: (
+          <FormattedMessage
+            id="xpack.ingestManager.agentConfigForm.nameFieldLabel"
+            defaultMessage="Name"
+          />
+        ),
+        placeholder: i18n.translate('xpack.ingestManager.agentConfigForm.nameFieldPlaceholder', {
+          defaultMessage: 'Choose a name',
+        }),
+      },
+      {
+        name: 'description',
+        label: (
+          <FormattedMessage
+            id="xpack.ingestManager.agentConfigForm.descriptionFieldLabel"
+            defaultMessage="Description"
+          />
+        ),
+        placeholder: i18n.translate(
+          'xpack.ingestManager.agentConfigForm.descriptionFieldPlaceholder',
+          {
+            defaultMessage: 'How will this configuration be used?',
+          }
+        ),
+      },
+    ];
+  }, []);
 
   return (
     <EuiForm>
-      {fields.map(({ name, label }) => {
+      {fields.map(({ name, label, placeholder }) => {
         return (
           <EuiFormRow
+            fullWidth
             key={name}
             label={label}
             error={touchedFields[name] && validation[name] ? validation[name] : null}
             isInvalid={Boolean(touchedFields[name] && validation[name])}
           >
             <EuiFieldText
+              fullWidth
               value={agentConfig[name]}
               onChange={e => updateAgentConfig({ [name]: e.target.value })}
               isInvalid={Boolean(touchedFields[name] && validation[name])}
               onBlur={() => setTouchedFields({ ...touchedFields, [name]: true })}
+              placeholder={placeholder}
             />
           </EuiFormRow>
         );
       })}
+      <EuiFormRow
+        label={
+          <EuiText size="xs" color="subdued">
+            <FormattedMessage
+              id="xpack.ingestManager.agentConfigForm.systemMonitoringFieldLabel"
+              defaultMessage="Optional"
+            />
+          </EuiText>
+        }
+      >
+        <EuiSwitch
+          showLabel={true}
+          label={
+            <>
+              <FormattedMessage
+                id="xpack.ingestManager.agentConfigForm.systemMonitoringText"
+                defaultMessage="Enable system monitoring"
+              />{' '}
+              <EuiIconTip
+                content={i18n.translate(
+                  'xpack.ingestManager.agentConfigForm.systemMonitoringTooltipText',
+                  {
+                    defaultMessage:
+                      'Enables system monitoring on this configuration by adding default data sources to it',
+                  }
+                )}
+                position="right"
+                type="iInCircle"
+              />
+            </>
+          }
+          checked={true}
+          onChange={() => {
+            // FIXME: need to implement Enable System Monitoring when API is available
```

Comment:
Can you please create and/or link to a ticket for this?

Reply:
Thanks @jfsiii . Will add reference here with issue.
I'm going to see if I can implement this on the API but may need more help in understanding what is needed. I will reach out on the slack team channel with question.

Reply:
Added link to issue tracking this. Thanks.